src/hotspot/share/runtime/mutexLocker.cpp

```diff
@@ -233,7 +233,7 @@ void mutex_init() {
   MUTEX_DEFN(Service_lock                    , PaddedMonitor, service);          // used for service thread operations
   MUTEX_DEFN(Notification_lock               , PaddedMonitor, service);          // used for notification thread operations
 
-  MUTEX_DEFN(JmethodIdCreation_lock          , PaddedMutex  , nosafepoint-2); // used for creating jmethodIDs.
+  MUTEX_DEFN(JmethodIdCreation_lock          , PaddedMutex  , nosafepoint-1); // used for creating jmethodIDs.
```

Comment:
Interesting. Why change from `nosafepoint-2` to `nosafepoint-1`?

Reply:
Found why it needed to be nosafepoint-1.  The DEFL macro is good for the global locks but HandshakeState_lock is not global since there's one per handshake operation.